### PR TITLE
add atlabs modules

### PR DESCRIPTION
**Pull request by i5dr0id**
#What does this PR do?'
- Add a module for Africa is talking

### CODE REVIEW

File: atlabs/airtime.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from atlabs.config import Atlabs
+from urllib.parse import urljoin
+from url import url as BASE_URL
+from .utils.constants import SEND_AIRTIME
+from .utils.request_handler import request
+
+
+class Airtime(Atlabs):
+
```
Comment (akinjide) [inline atlabs/airtime.py]: White spacing is inconsistent across this PR

File: atlabs/example/CreateCheckoutToken.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from atlabs.token import Token
+
+header = {
+    "base_url": "https://innovation-sandbox-backend.herokuapp.com",
+    "Sandbox-Key": "49264b2cc8fd68b33326c6d5468e5290",
```
Comment (akinjide) [inline atlabs/example/CreateCheckoutToken.py]: The examples should use random values and production URL

File: atlabs/example/CreatePremiumSubscription.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    "Sandbox-Key": "49264b2cc8fd68b33326c6d5468e5290",
+    "Content-Type": "application/json"
+}
+
+body = {
+    "shortCode": "19171",
+    "keyword": "innovation-sandbox",
+    "phoneNumber": "+2348123456789",
+    "checkoutToken": "CkTkn_65faa63e-cc95-41bb-812e-1c1d921df70b"
+}
```
Comment (akinjide) [inline atlabs/example/CreatePremiumSubscription.py]: The examples should use random values and production URL

File: atlabs/example/DeletePremiumSubscription.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    "base_url": "https://innovation-sandbox-backend.herokuapp.com",
+    "Sandbox-Key": "49264b2cc8fd68b33326c6d5468e5290",
+    "Content-Type": "application/json"
+}
+
+body = {
+    "shortCode": "19171",
+    "keyword": "innovation-sandbox",
+    "phoneNumber": "+2348123456789"
+}
```
Comment (akinjide) [inline atlabs/example/DeletePremiumSubscription.py]: The examples should use random values and production URL

File: atlabs/example/FetchMessage.py [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+header = {
+    "base_url": "https://innovation-sandbox-backend.herokuapp.com",
+    "Sandbox-Key": "49264b2cc8fd68b33326c6d5468e5290",
+    "Content-Type": "application/json"
+}
+
+body = {
+    "lastReceivedId": "0"
+}
```
Comment (akinjide) [inline atlabs/example/FetchMessage.py]: The examples should use random values and production URL

File: atlabs/example/FetchPremiumSubscription.py [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    "base_url": "https://innovation-sandbox-backend.herokuapp.com",
+    "Sandbox-Key": "49264b2cc8fd68b33326c6d5468e5290",
+    "Content-Type": "application/json"
+}
+
+body = {
+    "shortCode": "19171",
+    "keyword": "innovation-sandbox",
+    "lastReceivedId": "0"
+}
```
Comment (akinjide) [inline atlabs/example/FetchPremiumSubscription.py]: The examples should use random values and production URL

File: atlabs/example/MediaUpload.py [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+header = {
+    "base_url": "https://innovation-sandbox-backend.herokuapp.com",
+    "Sandbox-Key": "49264b2cc8fd68b33326c6d5468e5290",
+    "Content-Type": "application/json"
+}
+
+body = {
+    "phoneNumber": "+2348123456789",
+    "url": "http://test.com"
+}
```
Comment (akinjide) [inline atlabs/example/MediaUpload.py]: The examples should use random values and production URL

File: atlabs/example/QueueStatus.py [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+
+header = {
+    "base_url": "https://innovation-sandbox-backend.herokuapp.com",
+    "Sandbox-Key": "49264b2cc8fd68b33326c6d5468e5290",
+    "Content-Type": "application/json"
+}
+
+body = {
+    "phoneNumbers": "+2348123456789"
+}
```
Comment (akinjide) [inline atlabs/example/QueueStatus.py]: The examples should use random values and production URL

File: atlabs/example/SendAirtime.py [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+
+header = {
+    "base_url": "https://innovation-sandbox-backend.herokuapp.com",
+    "Sandbox-Key": "49264b2cc8fd68b33326c6d5468e5290",
+    "Content-Type": "application/json"
+}
+
+body = {
+    "recipients": [{"phoneNumber": "+2349091271976", "amount": "1000", "currencyCode": "NGN"}]
+}
```
Comment (akinjide) [inline atlabs/example/SendAirtime.py]: The examples should use random values and production URL

File: atlabs/example/SendPremiumMessage.py [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+body = {
+    "to": "+2348123456789",
+    "from": "FSI",
+    "message": "Hello world!",
+    "keyword": "innovation-sandbox",
+    "linkId": "d",
+    "retryDurationInHours": 1
+}
```
Comment (akinjide) [inline atlabs/example/SendPremiumMessage.py]: The examples should use random values and production URL

File: atlabs/utils/constants.py [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+
+SEND_AIRTIME = '/atlabs/airtime/send'
+
+VOICE_CALL = '/atlabs/voice/call'
+
+QUEUE_STATUS = '/atlabs/voice/queueStatus'
+
+MEDIA_UPLOAD = '/atlabs/voice/mediaUpload'
+
+CREATE_CHECKOUT_TOKEN = '/atlabs/token/checkout'
```
Comment (akinjide) [inline atlabs/utils/constants.py]: If constants are used once, might be better not declaring

File: atlabs/utils/response_handler.py [hunk 12]
```diff
@@ -0,0 +1,3 @@
+def response(data):
```
Comment (akinjide) [inline atlabs/utils/response_handler.py]: Remove unused code

File: sterling/transfer.py [hunk 13]
```diff
@@ -6,7 +6,8 @@
 
 class Transfer(Sterling):
     def InterbankNameEnquiry(self, query):
-        url = urljoin(BASE_URL(self.url), "/sterling/TransferAPIs/api/Spay/InterbankNameEnquiry")
+        url = urljoin(BASE_URL(self.url),
+                      "/sterling/TransferAPIs/api/Spay/InterbankNameEnquiry")
```
Comment (akinjide) [inline sterling/transfer.py]: What's the problem with having it oneline

File: sterling/account.py [hunk 14]
```diff
@@ -1,14 +1,14 @@
...
-from sterling.Sterling import  Sterling
+from sterling.Sterling import Sterling
 from sterling.utils.calls import calls
 
 
 class Account(Sterling):
     def InterbankTransferReq(self, body):
-        url = urljoin(BASE_URL(self.url), "/sterling/accountapi/api/Spay/InterbankTransferReq")
+        url = urljoin(BASE_URL(self.url),
+                      "/sterling/accountapi/api/Spay/InterbankTransferReq")
```
Comment (akinjide) [inline sterling/account.py]: Better to make the URL constant as you've done with atlab

File: atlabs/tests/common.py [hunk 15]
```diff
@@ -0,0 +1,160 @@
...
+
+header = {
+    "base_url": "",
+    "Sandbox-Key": str(Faker().text()),
+    "Content-Type": "application/json"
+}
+
+body = {
+    "CreateCheckoutToken": {
+        "phoneNumber": "+2349091271976"
```
Comment (akinjide) [inline atlabs/tests/common.py]: use faker for all test values

File: sterling/transfer.py (hunk 13 above)
Comment (i5dr0id) [reply sterling/transfer.py]: I'm using a linter

File: sterling/account.py (hunk 14 above)
Comment (i5dr0id) [reply sterling/account.py]: That would be in another PR

**Pull request merged**